### PR TITLE
ping opsgenie heartbeat

### PR DESCRIPTION
**Pull request by Mbaoma**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+    if not does_heartbeat_exist(CLUSTER_NAME, OPSGENIE_API_KEY):
+        logger.error(f"Heartbeat {CLUSTER_NAME} does not exist in Opsgenie.")
+    else:
+        logger.info(f"Heartbeat {CLUSTER_NAME} exist in Opsgenie.")
+
+        try:
+            while True:
+                ping_opsgenie_heartbeat()
+                time.sleep(PING_SLEEP)  # Sleep for 5 minutes #make an env
```
Comment (venkatamutyala) [inline main.py:87]: finish todo

File: __pycache__/json_log_formatter.cpython-311.pyc
Comment (venkatamutyala) [inline __pycache__/json_log_formatter.cpython-311.pyc:1]: This is a compiled file and shouldn't be checked in. Rebase the repo and remove it

File: main.py (hunk 1 above)
Comment (venkatamutyala) [reply main.py:87]: it looks like you already did this but didn't remove the comment

File: README.md [hunk 2]
```diff
@@ -1,2 +1,18 @@
...
+This repo contains a script that sends a ping to Opsgenie's Heartbeat every 5 minutes. If the cluster is down, Opsgenie will not get a ping and will send a mail to inform team members of the cluster's failed state.
+
+The script is deployed into the ArgoCD cluster under monitoring. Once this cluster is down, pings will not be sent to Opsgenie, triggering an alert which is sent to concerned team members.
+
+## Running the script
+
+- Create a ```.env``` file, with the following contents
+```bash
+OPSGENIE_API_KEY=<some-value>
+HEARTBEAT_NAME=<some-value>
```
Comment (venkatamutyala) [inline README.md:12]: missing variables?

**Review by venkatamutyala — CHANGES_REQUESTED** [review]

**Review by fernandoataoldotcom — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,2 +1,19 @@
...
+PING_SLEEP=<some-value>
+```
+
+- Runing the script
+```python
```
Comment (venkatamutyala) [inline README.md:17]: This isn't python. Not sure what the docs say but maybe there is a bash or a shell?

**Review by venkatamutyala — APPROVED** [review]

**Pull request merged**